students/Thai_H/lesson3/mail_room.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+import sys
+
+def send_thank_you_letter(donor_name, donation_amt):
+    print(f'\nDear {donor_name}\n'
+         'Thank you for a recent donation to our Foundation in the amount of $' + f'{donation_amt}\n'
+         '\nSincerely'
+         '\nBill & Melinda Gates')
+
+def is_existing_donor(donor_name):
+    tmp_donor_list = []
+    for each_donor in list_of_donors:
+        tmp_donor_list.append(each_donor[0])
+    return donor_name in tmp_donor_list # returns True if existing donor, False if new donor
```

Comment:
instead of looping through all of the donors, creating temp list and _then_ checking, check right away:
```
for donor in list_of_donors:
    if donor[0] == donor_name:
        return True
else:
    return False
```